src/Spice86.Core/Emulator/CPU/CfgCpu/Feeder/DiscriminatorReducer.cs

```diff
@@ -1,21 +1,22 @@
 namespace Spice86.Core.Emulator.CPU.CfgCpu.Feeder;
 
+using Spice86.Core.Emulator.CPU.CfgCpu.Linker;
 using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction;
 
 using System.Linq;
 
 public class DiscriminatorReducer {
-    private readonly IList<IInstructionReplacer<CfgInstruction>> _instructionReplacers;
+    private readonly List<IInstructionReplacer<CfgInstruction>> _instructionReplacers;
 
-    public DiscriminatorReducer(IList<IInstructionReplacer<CfgInstruction>> instructionReplacers) {
-        _instructionReplacers = instructionReplacers;
+    public DiscriminatorReducer(NodeLinker nodeLinker, InstructionsFeeder instructionsFeeder) {
```

Comment:
This really should take a list of instruction replacer objects, more may come.
This list could be handled at an upper level of course.

Reply:
Done.